packages/esm-ward-app/src/beds/occupied-bed.scss

```
@@ -0,0 +1,24 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@import '~@openmrs/esm-styleguide/src/vars';
```

```suggestion
@use '@openmrs/esm-styleguide/src/vars';
```

Requires `vars.$ui-02`, etc., but slightly friendlier to the Sass compiler.

---

I didn't know about this! Does it compile faster, or result in better output, or both?

@denniskigen should we add something about this to the docs, if we have docs on SCSS/style?

---

Theoretically, it compiles faster and produces more predictable output. However, the strongest reason is that the Sass team has said they're planning on deprecating `@import` altogether. [See here for more](https://sass-lang.com/documentation/at-rules/import/). (I say theoretically, because the `vars` file itself doesn't really add any bloat, but `@import` parts of Carbon definitely would).